api-server/src/main/java/vermillion/http/`

```diff
@@ -0,0 +1,1103 @@
+package vermillion.http;
```

Comment:
This file looks unnecessary. Please delete it.

Reply:
Deleted